src/ResourceTypeRegistry.js

```diff
@@ -36,6 +56,14 @@ export default class ResourceTypeRegistry {
     if(description) {
       this._resourceTypes[type] = {};
 
+      // Merge description defaults into provided description
+      description = merge({}, this._resourceDefaults, description);
```

Comment:
I'm fine with this as (and line 43) as the merge logic, and don't mind loading lodash :)